sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,15 +1,12 @@
 # Azure Form Recognizer client library for Python
 
-Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize text and table data
-from form documents. It includes the following main functionalities:
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to analyze text and structured data from your documents.
+It includes the following main features:
 
-* Custom models - Recognize field values and table data from forms. These models are trained with your own data, so they're tailored to your forms.
-* Content API - Recognize text, table structures, and selection marks, along with their bounding box coordinates, from documents. Corresponds to the REST service's Layout API.
-* Prebuilt models - Recognize data using the following prebuilt models
-    * Receipt model - Recognize data from sales receipts using a prebuilt model.
-    * Business card model - Recognize data from business cards using a prebuilt model.
-    * Invoice model - Recognize data from invoices using a prebuilt model.
-    * Identity document model - Recognize data from identity documents using a prebuilt model.
+* Layout - Extract text, table structures, and selection marks, along with their bounding region coordinates, from documents.
```

Comment:
Should Extract -> Analyze?

Reply:
for layout, I wanted to keep extract since it's does more of just an OCR read of the file.